wcomponents-core/src/main/java/com/github/bordertech/wcomponents/AbstractInput.java

```diff
@@ -38,6 +39,8 @@ public void addValidator(final FieldValidator validator) {
 
 		validator.setInputField(this);
 		model.validators.add(validator);
+		// possible source of memory leaks

```

Comment:
What do you think of calling the method `checkSize`. This indicates what we are `checking`. We dont have to describe the action in the method name as we may change from `logging` to `throw exception`.